Framework/Beamline/inc/MantidBeamline/DetectorInfo.h

```diff
@@ -134,6 +135,17 @@ inline size_t DetectorInfo::size() const {
   return m_isMonitor->size();
 }
 
+/** Returns the number of sum of the scan intervals for every detector in the
+ *instrument.
+ *
+ * If a detector is moving, i.e., has more than one associated position, every
+ *position is counted. */
+inline size_t DetectorInfo::scanSize() const {
```

Comment:
Did you observe performance issues if this is not inline? If not, prefer not inlining to avoid bloated headers?

Reply:
No, no performance issue, it was just to mimic the way similar methods were implemented. I un-inline this.